lib/broccoli/ember-app.js

```diff
@@ -895,6 +894,11 @@ class EmberApp {
     @return
   */
   _configReplacePatterns() {
+    let addons = this.project.addons;
```

Comment:
To clarify that these variables are only used for passing options to `contentFor` maybe you could tweak this to be an `options` POJO in the format passed below.  It _should_ reduce overall changes, and make it clearer that these are _only_ used for passing to `contentFor` util. 

What do you think?